cg/apps/sequencing_metrics_parser/parsers/utils.py

```diff
@@ -0,0 +1,3 @@
+def get_flow_cell_id(flow_cell_dir_name: str) -> str:
+    """Return the flow cell id from the flow cell directory name."""
+    return flow_cell_dir_name.split("_")[-1][1:]
```

Comment:
```suggestion
def get_flow_cell_id(flow_cell_dir_name: str) -> str:
    """Return the flow cell id from the flow cell directory name."""
    return flow_cell_dir_name.split("_")[-1][1:]
```
We have this functionality in the `FlowCellDirectoryData` we should not duplicate

Reply:
Yes, that is true.
I think it is unnecessary to create a `FlowCellDirectoryData` object just to extract the id though. Will place the utils function elsewhere and call it in the `FlowCellDirectoryData` class to avoid duplication 👍

Reply:
Hm, the exact logic does not exist in the `FlowCellDirectoryData`. I'll keep this as is

Reply:
I have seen that this function is needed in many places where we don't have a FlowCellDirectoryData available. I think is good to have this as an utils function.